collect_app/src/main/res/layout/server_auth_dialog.xml

```diff
@@ -61,11 +60,9 @@
         android:layout_marginLeft="20dip"
         android:layout_marginRight="20dip"
         android:scrollHorizontally="true"
-        android:inputType="textVisiblePassword"
-        android:autoText="false"
-        android:capitalize="none"
+        android:inputType="text|textNoSuggestions|textPassword"
+        android:textAllCaps="false"
```

Comment:
Do we really need that? Why? I guess you have added this instead of `android:capitalize="none"` ?
I think it's false by default and if we want to change something we can use:

> android:inputType="textCapCharacters|textCapSentences|textCapWords"

look here: https://developer.android.com/reference/android/widget/TextView.html#attr_android:capitalize am I wrong?